web_programming/get_top_billionaires.py

```diff
@@ -20,18 +20,29 @@
 )
 
 
-def calculate_age(unix_date: int) -> str:
+def calculate_age(unix_date: float) -> str:
```

Comment:
Why change the input to a float? Does `person["birthDate"]` store a float rather than an int?

Reply:
We divide unix_date by 1000, so the unix_date variable is redclared as float. 
The date passed is an int and we could just cast the division to an int but for simplicity sake I just type the param as float